docker/exporter/exporter.py

```diff
@@ -139,19 +140,13 @@ def main():
       '--bucket',
       help='Bucket name to export to',
       default=DEFAULT_EXPORT_BUCKET)
+  parser.add_argument('--ecosystem', required=True, help='Ecosystem to upload')
```

Comment:
If I'm understanding this correctly, the intended use of this flag is either an ecosystem name, or the special string "list", which retains the old behaviour? Please call out this "list" value in the flag help text.

Reply:
`list` value will upload the ecosystem.txt ecosystem list. Called this out in the help message now!